resolve/matrix/MatrixHandler.cpp

```diff
@@ -1,82 +1,98 @@
 #include <algorithm>
 
 #include <resolve/utilities/logger/Logger.hpp>
-#include <resolve/LinAlgWorkspace.hpp>
 #include <resolve/vector/Vector.hpp>
 #include <resolve/matrix/Coo.hpp>
 #include <resolve/matrix/Csc.hpp>
 #include <resolve/matrix/Csr.hpp>
+#include <resolve/workspace/LinAlgWorkspace.hpp>
+#include <resolve/utilities/misc/IndexValuePair.hpp>
 #include "MatrixHandler.hpp"
+#include "MatrixHandlerCpu.hpp"
+
+#ifdef RESOLVE_USE_CUDA
+#include "MatrixHandlerCuda.hpp"
+#endif
```

Comment:
```suggestion
#include "MatrixHandlerCpu.hpp"

#ifdef RESOLVE_USE_CUDA
#include "MatrixHandlerCuda.hpp"
#else
#include "MatrixHandlerCpu.hpp"
#endif
```
I assume we simply always want the CPU backend, so this suggestion is probably wrong.

Reply:
This is correct. We can always build CPU workspace, so we want to have matrix handler built, as well. Otherwise, you would not be able to run computations on CPU if you enable e.g. CUDA backend.